app/src/main/java/com/example/henzoshimada/feeltrip/ElasticSearchController.java

```diff
@@ -159,10 +195,13 @@ public GetMoodTask(String filterBy){
 
         @Override
         protected ArrayList<Mood> doInBackground(String... search_parameters) {
+            if(android.os.Debug.isDebuggerConnected())
+                android.os.Debug.waitForDebugger();
             verifySettings();
+
             ArrayList<Mood> moods = new ArrayList<>();
             String query;
-            if (fieldToSearch == null){
+            if (fieldToSearch == null){ //must specify what field you want to search upon creation of controller, should suffice for our purposes
```

Comment:
If need be, we can create another ControllerTask that handles the searching of any type with any field, but at the moment this bounded search should meet all of our purposes.